src/execution_plan/ops/op_create.c

```diff
@@ -133,7 +133,7 @@ static Record _handoff
 (
 	OpCreate *op
 ) {
-	return array_pop(op->records);
+	return (array_len(op->records)) ? array_pop(op->records) : NULL;
```

Comment:
The idea was to place a NULL as a marker
so you pop the last element which is NULL and we're done

Reply:
I wonder why the operation continues on asking for more data after it received `NULL`